### DISCUSSION TITLE
Brave Nightly v1.49.62 (Chromium 109.0.5414.119)

### DISCUSSION THREAD

**Discussion by daxbit** [community/community#119926] [Programming Help]
### Body

hi

i need a portable version from Brave Nightly v1.49.62 (Chromium 109.0.5414.119)

download here:
[https://github.com/brave/brave-browser/releases/download/v1.49.62/brave-...](https://github.com/brave/brave-browser/releases/download/v1.49.62/brave-v1.49.62-win32-x64.zip)

must be this version. newer version not work under windows 7.

this version from brave (download link) from github is half portable version. i can start it without instalation. but on other computer i los all my setting and synch code.

anyone here to can help me. to make this version full portable version?

i have gift, if a person can do it for me. i can give you a pcloud.com account (cloud storage) 500 GB for free

king regards
tony

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @daxbit,

Thanks for sharing your project with the community! Unfortunately, we currently do not allow self-promotion, advertising, or solicitation on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations.

Repeat posts may be considered a violation of our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and will result in a ban from the Community.

Thank you for helping us maintain a productive and tidy community for all our members.
